prov/verbs/src/ep_rdm/verbs_rdm_cm.c

```diff
@@ -615,21 +617,76 @@ ssize_t fi_ibv_rdm_conn_cleanup(struct fi_ibv_rdm_conn *conn)
 	return ret;
 }
 
+static int fi_ibv_rdm_poll_cq(struct fi_ibv_rdm_ep *ep)
```

Comment:
See `fi_ibv_rdm_tagged_poll`, it skips receive CQ polling if ep is closing. i think you may do the same for postponed processing.
BTW, all polling functions have out of date `tagged` prefix, it's used for all supported capabilities now.